jupyterhub/Chart.yaml

```diff
@@ -6,5 +6,6 @@ home: https://z2jh.jupyter.org
 sources:
   - https://github.com/jupyterhub/zero-to-jupyterhub-k8s
 icon: https://jupyter.org/assets/hublogo.svg
-kubeVersion: ">1.8.0"
-tillerVersion: ">2.7.0"
+# kubeVersion disabled awaiting fix to dash appendixed kube versions (-gke.1)
```

Comment:
I think you should just remove this rather than comment it out. We can always check back in git history :)

Reply:
Done! =)